### ISSUE TITLE
shellCommand.execute hangs in version v1.8.0

### ISSUE THREAD

**Issue by david-nagel** [augustocdias/vscode-shell-command#60]
I use shellCommand.execute in launch.json to set parameters for debug configurations.
After selecting the input provided by the command the 'Run and Debug' window shows the blue loading animation until I restart visual studio code
![image](https://user-images.githubusercontent.com/51096010/221403967-a594e2fc-5e89-4a14-a89b-457a28a74c17.png)

Downgrading to v1.7.5 fixes the issue

Using visual studio code 1.75.1 under Windows

```
        {
            "type": "command",
            "id": "targetPlatform",
            "command": "shellCommand.execute",
            "args": {
                "command": "ls ${workspaceFolder}/_products/Debug",
                "env": {
                    "PATH": "C:\\Program Files\\Git\\usr\\bin;${env:PATH}"
                },
                "useSingleResult": true
            }
        }
```

**Comment by mCr86** [comment]
The same for me.
1.8.0 stopped working.
I have downgraded it to 1.7.5 and it works again.

Manjaro 5.15.94-1
AUR visual-studio-code-bin 1.75.1-1

**Comment by augustocdias** [comment]
@quentingllmt could you help taking a look on this?

**Comment by augustocdias** [comment]
Can you check if v1.8.1 fixes the issue?
I've just published it

**Comment by mCr86** [comment]
Yes.
Solved.
Thanks!

**Comment by augustocdias** [comment]
Closed alongside #61
